frontstage/views/passwords/forgot_password.py

```diff
@@ -45,15 +51,22 @@ def post_forgot_password():
             if exc.status_code == 404:
                 logger.error('Requesting password change for email registered'
```

Comment:
@gavinleeedwards worth checking, but is this an actual error, maybe be like a warning log?

Reply:
also, if it is an `error` would we want to redirect?

Reply:
@shahi645 Good shout.  At this point it would be an actual error because the user should be in party.  This exception is also hit from a normal APIError - if party was down - so the log message wasn't covering everything.  As it's an error on our side, the user shouldn't be told that the email has been sent (they may have entered their email fine) so it will display the 500 page instead.